app/views/help/delegated-request.md

```diff
@@ -0,0 +1,5 @@
+<% title "How do I submit a delegated request on behalf of a card holder?" %>
```

Comment:
I've heard it referred to as "delegation" without the rest of the words, so wanted to give someone looking for that a cue in the question title. Doesn't show up elsewhere in the answer, though. Is it confusing or does it just seem wordy? (Latter seems like something to live with for the moment.)